pom.xml

```diff
@@ -15,44 +15,98 @@
   ~ limitations under the License.
   -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>com.arpnetworking.build</groupId>
+    <artifactId>arpnetworking-parent-pom</artifactId>
+    <version>1.0.2</version>
+    <relativePath/>
+  </parent>
   <modelVersion>4.0.0</modelVersion>
-  <name>Yammer Extra</name>
+  <name>Metrics Client Yammer Extras</name>
   <groupId>com.arpnetworking.metrics.extras</groupId>
-  <artifactId>yammer-extra</artifactId>
-  <version>0.3.4.GRPN.2-SNAPSHOT</version>
-  <packaging>jar</packaging>
+  <artifactId>metrics-yammer-extra</artifactId>
```

Comment:
The artifact id should remain yammer-extra. it is disambiguated given the group id is sub'd as metrics.extras. Thoughts?